collector/src/test/java/io/prometheus/jmx/TabularMXBean.java

```diff
@@ -0,0 +1,28 @@
+package io.prometheus.jmx;
+
+import javax.management.MBeanServer;
+import javax.management.ObjectName;
+import java.util.Collections;
+import java.util.Map;
+
+public interface TabularMXBean {
+    Map<String, String> getTable();
+
+    int getFoo();
+}
+
+class Tabular implements TabularMXBean {
+    public static void registerBean(MBeanServer mbs) throws javax.management.JMException {
+        ObjectName mbeanName = new ObjectName("tabular:type=Test");
+        Tabular mbean = new Tabular();
+        mbs.registerMBean(mbean, mbeanName);
+    }
+
+    public Map<String, String> getTable() {
+        return Collections.singletonMap(null, "null key");
```

Comment:
This seems to be testing a null key rather than a null value

Reply:
Yes. It causes error, not null value.

Reply:
I'm not understanding, I don't see how this tests the code change.